### ISSUE TITLE
lesion_merged.nii.gz shape does not match MNI-Poly-AMU shape  

### ISSUE THREAD

**Issue by jcohenadad** [sct-pipeline/cord-erosion#4]
~~~
/MS088_R01_20 $ fslhd lesion_merged.nii.gz 
filename	lesion_merged.nii.gz
size of header	348
data_type	UINT16
dim0		3
dim1		141
dim2		141
dim3		1100
~~~

MNI-Poly-AMU from SCT v3.0.1 (https://osf.io/sh6h4/)
~~~
MNI-Poly-AMU/template $ fslhd MNI-Poly-AMU_cord.nii.gz 
filename	MNI-Poly-AMU_cord.nii.gz
size of header	348
data_type	UINT8
dim0		3
dim1		120
dim2		120
dim3		600
~~~

@RAOuelletteIV Can you explain this discrepancy?

**Comment by jcohenadad** [comment]
Digging a bit more, I noticed that the PAM50 from SCT v3.0.1 has the matching dims:
~~~
PAM50/template $ fslhd PAM50_cord.nii.gz 
filename	PAM50_cord.nii.gz
size of header	348
data_type	FLOAT32
dim0		3
dim1		141
dim2		141
dim3		1100
~~~
@RAOuelletteIV Do you confirm you've been using the PAM50 template?

**Comment by RAOuelletteIV** [comment]
Exactly yes, the merging script I have uses the PAM50 template, for the csa_wm/gm and then also for lesions if it exists. The specific commands are as follows. 
~~~
  sct_merge_images -i ${input_csa_wm} -w ${input_warp_csa} -d $SCT_DIR/data/PAM50/template/PAM50_t2.nii.gz -o csa_wm_merged.nii.gz -x nn
  sct_merge_images -i ${input_csa_gm} -w ${input_warp_csa} -d $SCT_DIR/data/PAM50/template/PAM50_t2.nii.gz -o csa_gm_merged.nii.gz -x nn
  if [ ! -z "$input_lesion" ]; then
    sct_merge_images -i ${input_lesion} -w ${input_warp_lesion} -d $SCT_DIR/data/PAM50/template/PAM50_t2.nii.gz -o lesion_merged.nii.gz -x linear
  fi
~~~

Great catch thank you! For the manuscript, regarding GM and WM segmentation, should I replace "_Dupont SM, De Leener B, Taso M, Le Troter A, Nadeau S, Stikov N, et al. Fully-integrated framework for the segmentation and registration of the spinal cord white and gray matter. NeuroImage 2017; 150: 358–372._" with "_De Leener B, Fonov VS, Collins DL, Callot V, Stikov N, Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space. NeuroImage 2018; 165: 170–179._"?

**Comment by jcohenadad** [comment]
> For the manuscript, regarding GM and WM segmentation, should I replace "_Dupont SM, De Leener B, Taso M, Le Troter A, Nadeau S, Stikov N, et al. Fully-integrated framework for the segmentation and registration of the spinal cord white and gray matter. NeuroImage 2017; 150: 358–372._" with "_De Leener B, Fonov VS, Collins DL, Callot V, Stikov N, Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space. NeuroImage 2018; 165: 170–179._"?

No, these are two different things. Text should be something like: each individual data was registered to the PAM50 template [De Leener NIMG 2018] using the method described in [Dupont NIMG 2017].

**Comment by RAOuelletteIV** [comment]
The script worked great! I applied it to the entire cohort and am now processing the results. I had to combine the innermost layers 7-9, so that the layers were more comparable and represented through the majority of the length of the spinal cord.

**Comment by jcohenadad** [comment]
> I had to combine the innermost layers 7-9

Cool! I suggest you push whatever changes you made to the script in this branch, so that you can include the github link in your manuscript for full transparency (we can do a release to freeze the code in time).
